Description: Get number from getmodemspeed.
 From debian/changelog for version 1.3.0-1:
 .
 Patched wmppp/wmppp.c to expect a NUMBER from getmodemspeed, and not the
 whole connect line.
 .
 Converted to quilt for version 1.3.0+20140915-1.  Also commented out
 declaration of p to avoid unused variable compiler warning.
Author: Marcelo E. Magallon <mmagallo@debian.org>
Author: Doug Torrance <dtorrance@monmouthcollege.edu>
Last-Update: 2015-05-20

--- a/wmppp.c
+++ b/wmppp.c
@@ -847,7 +847,7 @@
 	fp = popen(speed_action, "r");
 
 	if (fp) {
-		char *p;
+		/* char *p; */
 		char temp[128];
 		int i, linespeed;
 
@@ -858,9 +858,12 @@
 
 		pclose(fp);
 
+#if 0
 		if ((p=strstr(temp, "CONNECT"))) {
 			linespeed = atoi(p + 8);
 		}
+#endif
+		linespeed = atoi(temp);
 
 		k = 30;
 
